sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -71,7 +73,7 @@ describe("New session token", function () {
 });
 
 // For some reason this test does not pass against the emulator. Skipping it for now
-describe.skip("Session Token", function (this: Suite) {
+describe.only("Session Token", function (this: Suite) {
```

Comment:
Nice, is this test passing now? Or what's the original issue that's failing this test?

Reply:
I think this is skip because it requires the emulator to pass.

Reply:
✓ retries session not found successfully (2401ms)
✓ Should check if the MaxIntegratedCacheStaleness header is set. (2329ms)

but the full test passes with the emulator loaded.